### ISSUE TITLE
New app icon

### ISSUE THREAD

**Issue by syt0r** [SYtor/Kanji-Dojo#170]
Current icon is quite plain and not very distinct, making it easy to confuse Kanji Dojo with other apps. I wish to create a better icon but my art skills are no good 😓  

If anyone is willing to help with new [adaptive vector icon](https://developer.android.com/develop/ui/views/launch/icon_design_adaptive) I would appreciate it very much. I could add your credit to the app's about screen with url and mention it in GitHub's readme

The icon has to be minimalistic and showcase something relevant, like some kanji or word, japan, training dojo or similar. Additional input and ideas are appreciated too

**Comment by ThomasLeigh** [comment]
Good point that the icon should distinguish the app from all the rest - especially given the impression that Kanji are used (way too) often for icons (or even logos) for various Japan-related things. This results in lack of brand identity.

Instead, perhaps a good idea is to consider things related to the Japanese tradition: key symbols related to shodou (calligraphy) or ancient writing techniques / tools. I think that such an approach could definitely add an original flavour for the app's identity.

Look, for example, at this icon (from [the set](https://www.deviantart.com/dunedhel/art/Pack-Yuuyake-96029071)):

![Gimp.png](https://github.com/user-attachments/assets/f9d5b813-7387-4e73-bb7e-6e3e95dda1e0)

It's very sad and depressing that I cannot help in this matter, because, although I'm a graphic designer, currently I don't have a computer and I'm not in position to gain another one :( . So I can only keep fingers crossed that a new icon emerges in its glory :) .

**Comment by syt0r** [comment]
Interesting idea, I might look into some traditional tools too

Up to now I had ideas like a bamboo shoot (cutting?) and bonsai. These things are widely known so many learners could get a sense of familiarity from them as well as it can give off some ideas of practice/growth but they might be considered very common as well. I think of making it gray-black with some small detail accented with red color

I also tried to create some abstract icon using simple rectangular shapes for cards and dots colored like new, due, done indicators from the app, trying to make it look like flashcards with SRS statuses, it ended up looking similar to Todai app icons but it didn't look that well

I've been saving icons I like for some time to use for references, overall there are so many cool and creative icons for japanese apps, even something generic and simple can looks good and distinctive too

<img width="200" alt="image" src="https://github.com/user-attachments/assets/3128c735-7459-4e8e-9cd0-76a3f9ea645a">
<img width="300" alt="image" src="https://github.com/user-attachments/assets/d605a616-4e03-4441-bd22-5f416b9b3be2">
<img width="250" alt="image" src="https://github.com/user-attachments/assets/45ec7989-0ba1-458c-9ba1-74a2074a6e3f">
<img width="400" alt="image" src="https://github.com/user-attachments/assets/5a64ff25-fc72-469a-958c-b87ae16875f7">
<img width="200" alt="image" src="https://github.com/user-attachments/assets/64ee157e-9104-4cee-b312-bad6046e242f">

**Comment by neithernut** [comment]
Over the last few days I spend some time playing around with logo ideas. My drawing abilities are insufficient (I'm not an illustrator), and I have no experience in doing adaptive vector graphics. But maybe someone else wants to pick this up.

The first idea was to base the logo on a sitting person doing some calligraphy, like [this](https://commons.wikimedia.org/wiki/File:Billet-Doux_-_Japanese_woman_writing_a_letter_on_paper_roll_with_an_ink_brush_(1911_by_Elstner_Hilton).jpg) or [this](https://commons.wikimedia.org/wiki/File:Furyu-no_tsuki_(BM_2008,3037.20401_1).jpg). Making the brush recognizable would probably not be that easy.

The second idea was to focus on the "dojo" in the app's name. Most (western) users will probably associate "dojo" with martial arts dojos, so I sketched a person doing sword swings with an oversized brush. I don't think this will work as a logo: the level of detail required is likely too high.

**Comment by ThomasLeigh** [comment]
Today's experiments:
![Image](https://github.com/user-attachments/assets/ce5166af-ebff-4606-83e6-826e57337029)

**Comment by syt0r** [comment]
I appreciate your efforts, but the style is quite different to what I have in mind

**Comment by omegaui** [comment]
@syt0r 
It's been 2 month since last comment
Have you figured something or still looking for one?
I've experience in both professional app designing and I'm a flutter developer by profession working as a head of development team at BillingFast, India.

I'm learning japanese using your app, and I've cleared several N5 mock exams, currently practising for N4.

Let's discuss the color palette you have in mind for the icon also some details about the art style if you want a new icon.

As a modern app, the icon should follow fluency, material or nimbus design patterns.
